v5/v5_stable.ino

```diff
@@ -1868,12 +1869,19 @@ bool hasPackage() {
   } else {
     Println(7, "Set Time First!");
   }
-  setField_MonthAndDate(package_expiry_date, expiryDate, expiryMonth);
-  if (RTC.month <= expiryMonth)
+  setField_MonthAndDate(package_expiry_date, expiryMonth, expiryDate);
+  if (RTC.month <= expiryMonth && RTC.month != 0) {
```

Comment:
seems like this was the main logical issue! right?
at start they both are equal to zero and this causes issue.

Reply:
yes, you are right.